src/tests/build.proj

```diff
@@ -214,6 +214,22 @@
       <TestTargetingPathLibs Include="$(TargetingPackPath)/*.dll" />
     </ItemGroup>
 
+    <!-- common linker arguments for appbuilder -->
+    <ItemGroup>
+      <_CommonLinkerArgs Include="libz.so" />
+      <_CommonLinkerArgs Include="log" />
+      <_CommonLinkerArgs Include="-u GlobalizationNative_LoadICU" />
```

Comment:
ugh, even more duplication... I feel like this needs addressing somehow or we'll end up in a maintenance mess.

Reply:
I can throw them into some common.props that we can import.

Reply:
I ended up moving them back into the cmake template.